src/ssz/type/vector.zig

```diff
@@ -48,6 +52,10 @@ pub fn FixedVectorType(comptime ST: type, comptime _length: comptime_int) type {
             try merkleize(@ptrCast(&chunks), chunk_depth, out);
         }
 
+        pub fn clone(value: *const Type, out: *Type) !void {
+            for (0..length) |i| try Element.clone(&value[i], &out[i]);
```

Comment:
can the implementation here match the one in byte_vector?

Reply:
good catch, yeah i think this is doable for the fixed types, simplifies implementation quite a bit. Done in [1898a85](https://github.com/ChainSafe/ssz-z/pull/51/commits/1898a85062cc0f3610ca7e3069a92dda160bb6af)